src/blockchain/blockchain_parameters.h

```diff
@@ -171,9 +171,19 @@ struct Parameters {
   //! \brief The base block reward
   CAmount reward;
 
-  //! \brief The reward immediately given upon block proposal.
+  //! \brief The fraction of the reward immediately given upon block proposal.
   ufp64::ufp64_t immediate_reward_fraction;
 
+  //! \brief The fraction of the reward given to the proposers after finalization.
+  //!
+  //! Proposers receive their reward in two parts. The first part is paid
+  //! immediately in the coinbase transaction of the proposed block. The second
+  //! part is paid in the first block of the following epoch. The amount of the
```

Comment:
> The second part is paid in the first block of the following epoch

I think this should be rephrased if the second part comes after finalization, as we could enter into a new epoch without having finalization.

Reply:
The second part comes regardless of the finalization. But its size will depend on the number of included votes.